tests/conftest.py

```diff
@@ -5,9 +5,15 @@
 import pythoncom
 import nitsm.codemoduleapi
 
-_standalone_tsm_context_tlb = win32com.client.selecttlb.FindTlbsWithDescription(
-    "NI TestStand Semiconductor Module Standalone Semiconductor Module Context"
-)[0]
+try:
+    _standalone_tsm_context_tlb = win32com.client.selecttlb.FindTlbsWithDescription(
+        "NI TestStand Semiconductor Module Standalone Semiconductor Module Context"
+    )[0]
+except IndexError:
+    raise IndexError(
+        """The TSM Standalone Semiconductor Module Context component is not installed. 
+                        Contact one of the repository owners to determine how to obtain this non-public component."""
```

Comment:
For a multi-line string like this the whitespace is maintained.  

![image](https://user-images.githubusercontent.com/51342134/139674448-16ac8212-fbc6-4e24-aa6c-bdd4ddc83d63.png)

If it's too much to fit in 100 chars you can do something like this to break it up.
![image](https://user-images.githubusercontent.com/51342134/139674649-ef9bd724-4554-465b-ad2b-c71589c825f3.png)

![image](https://user-images.githubusercontent.com/51342134/139674676-91c1b999-6f4a-4d06-93a2-450ca97252c3.png)